src/_breakpoints.scss

```
@@ -207,3 +207,83 @@
     @content;
   }
 }
+
+/// Standard breakpoint to support resolutions greater than 1232px.
+/// @example scss - import mixin file
+///   @import "./node_modules/@aurodesignsystem/webcorestylesheets/dist/grid-breakpoints";
+/// @group responsive
+/// @example scss - Set grid-breakpoint
+///   .foo {
+///       @include auro_grid-breakpoint--xl {
+///         ...
+///       }
+///   }
+@mixin auro_grid-breakpoint--xl {
+  @media screen and (min-width: $auro-grid-breakpoint-xl) {
```

One thing I want to do now that CSS variables are used in all browsers is to STOP using Sass variables. 

```suggestion
  @media screen and (min-width: var(--auro-grid-breakpoint-xl) {
```

By doing this, we can remove the hard-coded expectation and not inadvertently break a test just because the value of a variable changed.

---

Replaced the use of Sass variables with CSS variables in the media query mixins as well as their unit tests.